### DISCUSSION TITLE
UserWarning: Empty or unsupported dataset attached to actor

### DISCUSSION THREAD

**Discussion by giulianogrossi** [pyvista/pyvista#2747] [Q&A]
I am working on notebook by activating `pv.set_jupyter_backend('pythreejs')`.
I am trying to show labels of the mesh points but all the examples I try (taken from the documentation and Official Examples - for example that one [add_point_labels](https://docs.pyvista.org/api/plotting/_autosummary/pyvista.Plotter.add_point_labels.html)) show the following warning 
```
miniconda3/lib/python3.9/site-packages/pyvista/jupyter/pv_pythreejs.py:436: UserWarning: Empty or unsupported dataset attached to actor
  warnings.warn('Empty or unsupported dataset attached to actor')
```
and shows no label in the image. Would anyone kindly have an idea why this could happen systematically?

Thank you!

Giuliano

**Comment by akaszynski** [comment]
There's a solution for this in https://github.com/pyvista/pyvista/pull/3102. But for now, `pythreejs` doesn't support labes. I'm strongly considering writing a `pyvista-k3d` module to port our plotter to `k3d` since it's become quite mature over the past few years.
